limelight.py

```diff
@@ -0,0 +1,142 @@
+import cv2
+import numpy as np
+import imutils
+
+# Camera Constants
+KNOWN_DIAMETER_IN = 9.5  # in
+# ADJ_FOCAL_LENGTH = 1480.1
+FOCAL_LENGTH = 1367.043233  # 83.3% accuracy. Use 1645.101 -> 99.73% accuracy but overshoots
+RADIUS_THRESH = 85
+MIN_BORDERS = 6
+CAMERA_TILT_DOWNWARDS = 20
+
+
+# CV Input Code
+def set_up_camera(camera_port):
+    camera_path = "/dev/video" + str(camera_port)
+
+    # set the width and height property of the image
+    live_input.set(cv2.CAP_PROP_FRAME_WIDTH, 1920)
```

Comment:
We don't need any camera setup, LimeLight does camera setup through the UI under Input tab.

Reply:
Thank you for the feedback. I have removed that part.